src/helpers/xml-builder.js

```diff
@@ -349,9 +397,9 @@ const cssBorderParser = (borderString) => {
       ].includes(token)
     ) {
       // Accepted OOXML Values for border style: http://officeopenxml.com/WPtableBorders.php
-      if (['dashed', 'dotted', 'double', 'inset', 'outset'].includes(token)) {
+      if (['dashed', 'dotted', 'double', 'inset', 'outset', 'none'].includes(token)) {
         stroke = token;
-      } else if (['hidden', 'none'].includes(token)) {
+      } else if (token === 'hidden') {
```

Comment:
So if you provide hidden to a table border or table cell border then no matter what other properties are given, the borders will remain hidden.
Example: If you set for table, border-style as hidden and then to a table cell provide border-style as solid, no border will be shown for outer cells(i.e. no table border)

Now, in code we needed to distinguish it because if we provide none instead of hidden then the values are overwritten.